lib/inputstreamhelper/__init__.py

```diff
@@ -546,6 +546,61 @@ def _chromeos_config(self):
 
         return devices
 
+    def _backup_libwv(self):
+        """Creates backup of installed lib, keeping multiple backups according to setting 'backups'"""
+        backups = int(get_setting('backups', '2'))
+        bak_dir = os.path.join(self._ia_cdm_path(), 'backup')
```

Comment:
I would go for backup_path :-)

Reply:
In my opinion it should be dir, because it is a directory. path could be both, file or dir.

Reply:
I changed bak to backup, but kept the dir.